spec/forms/user_piv_cac_setup_form_spec.rb

```diff
@@ -103,6 +103,48 @@
       end
     end
 
+    context 'when piv cac is required' do
+      let(:token) { 'good-token' }
+      let(:x509_dn_uuid) { 'random-uuid-for-x509-subject' }
+      let(:form) do
+        described_class.new(user: user, token: token, nonce: nonce, name: 'Card 1',
+                            piv_cac_required: true)
+      end
+      let(:no_eku_token_response) do
+        { 'nonce' => nonce, 'key_id' => 'foo', 'has_eku' => false, 'uuid' => x509_dn_uuid,
+          'subject' => 'x509-subject' }
+      end
+      let(:has_eku_token_response) do
+        { 'nonce' => nonce, 'key_id' => 'foo', 'has_eku' => true, 'uuid' => x509_dn_uuid,
+          'subject' => 'x509-subject' }
+      end
```

Comment:
I would just make a new `let` for `has_eku` since the rest of these are the same? then we can override that in a specific context?